msticpy/nbtools/security_alert.py

```diff
@@ -119,47 +125,16 @@ def _resolve_entity_refs(self):
             if not isinstance(entity, Entity):
                 continue
             for prop_name, prop_val in entity.properties.items():
-                if isinstance(prop_val, dict) and '$ref' in prop_val:
-                    entity_id = prop_val['$ref']
+                if isinstance(prop_val, dict) and "$ref" in prop_val:
+                    entity_id = prop_val["$ref"]
                     if entity_id in self._src_entities:
                         entity[prop_name] = self._src_entities[entity_id]
 
-    def _find_os_family(self):
```

Comment:
You still call this (Line 52) in the file despite deleting the function.

Reply:
True but it should now call the version in the SecurityBase class instead of the redundant local copy.